pkg/auth/service.go

```diff
@@ -582,9 +582,10 @@ func (s *DBAuthService) ListGroupUsers(ctx context.Context, groupDisplayName str
 			WHERE
 				auth_groups.display_name = $1
 				AND auth_users.display_name > $2
-			ORDER BY auth_users.display_name
-			LIMIT $3`,
-			groupDisplayName, params.After, params.Amount+1)
+				AND auth_users.display_name like $3
```

Comment:
why listing needs to filter by prefix? we use this one as `after` which means you should get something that doesn't match the value's prefix.

Reply:
All good - didn't saw it is prefix.